tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -529,6 +532,209 @@ def increment(self, event, count, environment=None, timestamp=None):
             )
 
 
+@apply_feature_flag_on_cls("organizations:event-unique-user-frequency-condition-with-conditions")
+class EventUniqueUserFrequencyConditionWithConditionsTestCase(StandardIntervalTestBase):
+    __test__ = Abstract(__module__, __qualname__)
+
+    rule_cls = EventUniqueUserFrequencyConditionWithConditions
+
+    def increment(self, event, count, environment=None, timestamp=None):
+        timestamp = timestamp if timestamp else before_now(minutes=1)
+        data = {"fingerprint": event.data["fingerprint"]}
+        if environment:
+            data["environment"] = environment
+
+        for _ in range(count):
+            event_data = deepcopy(data)
+            event_data["user"] = {"id": uuid4().hex}
+            self.add_event(
+                data=event_data,
+                project_id=self.project.id,
+                timestamp=timestamp,
+            )
+
+    def test_comparison(self):
+        pass
+
+    def test_comparison_empty_comparison_period(self):
+        pass
```

Comment:
Why are we skipping these?

Reply:
added these cases in. i'm not sure that they test any different behavior for this condition but i've added them regardless.